src/shared/store/index.ts

```diff
@@ -183,18 +185,18 @@ export function transactionsList(): InternalEconomyTransaction[] {
 //
 // Draft utility functions
 //
-export function getDraft(id: string): InternalDraft | undefined {
-  if (!id || !globalStore.drafts[id]) return undefined;
-  return globalStore.drafts[id];
+export function getDraft(id: string): InternalDraftv2 | undefined {
+  if (!id || !globalStore.draftsv2[id]) return undefined;
+  return globalStore.draftsv2[id];
 }
 
 export function draftExists(id: string): boolean {
-  return globalStore.drafts[id] ? true : false;
+  return globalStore.draftsv2[id] ? true : false;
```

Comment:
```suggestion
return globalStore.draftsv2[id] !== undefined;
```
Not sure if it can be null, too, though.

Reply:
not sure, but it doesnt hurt to leave it ?

Reply:
No it doesn't. It's just that `return condition ? true : false;` always look funny to me, when you could just `return condition;` ;)